components/Dicom/dwv.vue

```diff
@@ -225,6 +243,50 @@ export default {
       loadFromOrthanc: false,
       dicom: [],
       instanceNumber: 1,
+      helpInfo: [
+        {
+          name: "*Scroll",
+          info: "Toggle instances",
+          use: [
+            "Mouse wheel scrolls over the image",
+            "Mouse click on the image to drag up and down",
+          ],
+        },
+        {
+          name: "*ZoomAndPan",
+          info: "Drag, zoom in and out",
+          use: [
+            "Mouse wheel scrolls over the image",
+            "Mouse click on the image to drag it",
+          ],
+        },
+        {
+          name: "*WindowLevel",
+          info: "Toggle Contrast",
+          use: ["Mouse click on the image to drag up, down, left and right"],
+        },
+        {
+          name: "Binder",
+          info: "Active in MPR mode",
+          use: ["Select WindowLevel/Position/Zoom/Offset/Opacity"],
```

Comment:
Only changes with checked options will apply to all three panels. Changes will only applied to the primary panel if the associated option is unchecked.